### ISSUE TITLE
✅ 🧩 🐞 Taxonomy Filter - Item CSS 'Cursor' Not Displaying Correctly - [ED-14522]

### ISSUE THREAD

**Issue by anakeme** [elementor/elementor#24322]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

The **Taxonomy Filter** items show that the CSS contains `cursor: pointer`, however it does not display on hover unless I setup a CSS code and class it as `important;`.

This occurs when used with a **Loop Grid** using **Advanced Custom Fields (ACF)** as the Taxonomy Filter.

I am unable to re-create this without the 3rd party plugin as it contains both the Taxonomy and custom Post Type for the items I am filtering. However, I did test this using the inbuilt WordPress Posts type and setup a test - it did not work for that either.

### Steps to reproduce

1) Create Loop Grid.
2) Create Taxonomy Filter for Loop Grid.

### Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>Click to reveal</summary>

````txt



== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: Source distribution v5.7.39-42
	PHP Version: 7.4.33
	PHP Memory Limit: 768M
	PHP Max Input Vars: 3000
	PHP Max Post Size: 256M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.4.1
	Site URL: https://jessem81.sg-host.com
	Home URL: https://jessem81.sg-host.com
	WP Multisite: No
	Max Upload Size: 256 MB
	Memory limit: 256M
	Max Memory limit: 768M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: 10
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.9.0
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/119.0.0.0 Safari/537.36 Edg/119.0.0.0

== Active Plugins ==
	Advanced Custom Fields
		Version: 6.2.3
		Author: WP Engine

	Code Snippets
		Version: 3.6.2
		Author: Code Snippets Pro

	Elementor
		Version: 3.17.3
		Author: Elementor.com

	Elementor Header &amp; Footer Builder
		Version: 1.6.17
		Author: Brainstorm Force, Nikhil Chavan

	Elementor Pro
		Version: 3.17.1
		Author: Elementor.com

	Security Optimizer
		Version: 1.4.7
		Author: SiteGround

	Speed Optimizer
		Version: 7.4.3
		Author: SiteGround


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	


== Elementor Experiments ==
	Optimized DOM Output: Active by default
	Improved Asset Loading: Active by default
	Improved CSS Loading: Active by default
	Inline Font Icons: Inactive by default
	Additional Custom Breakpoints: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active by default
	Upgrade Swiper Library: Active by default
	Grid Container: Active
	Default to New Theme Builder: Active by default
	Hello Theme Header &amp; Footer: Active by default
	Editor Top Bar: Inactive by default
	Optimized Gutenberg Loading: Active by default
	Build with AI: Inactive by default
	Landing Pages: Active by default
	Nested Elements: Active
	Lazy Load Background Images: Inactive by default
	Optimize Image Loading: Active by default
	Global Style Guide: Active by default
	Page Transitions: Inactive
	Notes: Active by default
	Form Submissions: Active by default
	Scroll Snap: Active by default
	Menu: Active
	Taxonomy Filter: Active


== Log ==
	
JS: showing 3 of 3JS: 2023-11-16 07:20:53 [error X 3][../wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.17.3:3:890462] Cannot read properties of undefined (reading &#039;route&#039;) 
JS: 2023-11-16 10:29:28 [error X 2][../wp-content/plugins/elementor-pro/assets/js/gallery.8ca9a354ce039d1ba641.bundle.min.js:2:5674] Cannot read properties of undefined (reading &#039;setSettings&#039;) 
JS: 2023-11-17 10:41:00 [error X 2][https://jessem81.sg-host.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.17.3:3:639997] Cannot read properties of undefined (reading &#039;isDesignable&#039;) 

PHP: showing 3 of 3PHP: 2023-11-16 20:13:01 [notice X 38][../wp-content/plugins/elementor-pro/modules/dynamic-tags/acf/tags/acf-text.php::33] Undefined offset: 1 [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-11-16 21:19:17 [error X 4][../wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::312] Uncaught Error: Call to a member function update_meta() on null in ../wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php:312
Stack trace:
#0 ../wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php(185): ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager-&gt;save_conditions(6, Array)
#1 [internal function]: ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager-&gt;ajax_save_theme_template_conditions(Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#2 ../wp-content/plugins/elementor/core/common/modules/ajax/module.php(176): call_user_func(Array, Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#3 ../wp-includes/class-wp-hook.php(324): Elementor\Core\Common\Modules\Ajax\Module-&gt;handle_ajax_request(&#039;&#039;)
#4 /home/customer/www/j [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-11-17 19:10:08 [notice X 9][../wp-content/plugins/elementor-pro/modules/carousel/widgets/base.php::573] Trying to access array offset on value of type null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]



== Elementor - Compatibility Tag ==
	
	Elementor Header & Footer Builder: Compatible
	Elementor Pro: Compatible

== Elementor Pro - Compatibility Tag ==
	
	Elementor Header & Footer Builder: Compatible


````

</details>

**Comment by KingGrizzly** [comment]
I get the same behavior. No cursor by default.

**Comment by contrastxxx** [comment]
same problem

**Comment by Johnvrkmp** [comment]
same thing

**Comment by bperry420** [comment]
I have the same problem, only I am NOT using ACF (just elementor)

**Comment by nicholaszein** [comment]
Hey everyone!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro v3.22.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
